src/Components/EditWarehouse/EditWarehouse.jsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from "react";
+import "./EditWarehouse.scss";
```

Comment:
Your ./EditWarehouse.scss file is empty but styls are still being applied to this page cant see where they are coming from

Reply:
and also check you mobile responsiveness , its fine  on desktop and tablet